resources/publish.rb

```diff
@@ -24,13 +24,18 @@
 property :prefix,        String, default: ''
 property :distribution,  String, default: ''
 property :timeout,       Integer, default: 3600
+# If set to anything non-empty, used as GPG passphrase instead of the node
+# attribute. This is useful if you did something like replacing the original GPG
+# key created by the recipe.
+property :gpg_passphrase, String, default: ''
```

Comment:
why not make the default the value of: `node['aptly']['gpg']['passphrase']`

Reply:
Something something node attributes?

Reply:
I thought I left a comment here half a day ago but apparently not.

That default does seems like what should be there in the first place, it was an oversight.

I pushed an update an re-requested a review, PTAL